.github/workflows/cpack.yml

```diff
@@ -69,11 +69,28 @@ jobs:
           chmod +x /root/git.sh
           update-alternatives --install /usr/bin/git git /root/git.sh 50
 
+      # /github/workspace is added to "safe" dirs in global config e.g.
+      # ~/.gitconfig so both runner and builder containers trust these dirs
+      # owned by different UIDs from that of Git's EUID. This is made necessary
+      # by newly-enforced directory boundaries in Git v2.35.2
+      # ref: https://lore.kernel.org/git/xmqqv8veb5i6.fsf@gitster.g/
+      - name: declare to Git "safe" directories owned by a different UID
+        run: |
+          for dir in /github/workspace /__w/ziti-tunnel-sdk-c/ziti-tunnel-sdk-c; do
```

Comment:
what is `for` loop for?

Reply:
An earlier version ran multiple commands for multiple dirs, and now runs one command for multiple dirs (space-separated on line 79). This could be rewritten to make it clearer to the eye there are multiple iterations here.